avocado/utils/cpu.py

```diff
@@ -71,7 +71,8 @@ def get_cpu_vendor_name():
     """
     Get the current cpu vendor name
 
-    :returns: string 'intel' or 'amd' or 'power7' depending on the current CPU architecture.
+    :returns: string 'intel' or 'amd' or 'power7' depending on the
+        current CPU architecture.
```

Comment:
Could you please split the commits? It's not recommended to bound style-fixes and new code in one commit.

_... maybe a silly advice, but take a look at `git citool` (on fedora part of `git-gui` package) or `git cola`, both let you graphically amend patches and easily pick the section you want to include/save for later)._